backends/ionicsqlite.js

```diff
@@ -0,0 +1,58 @@
+module.exports = class IonicSqlite{
+    constructor() {
+
+    }
+
+    addOne(doctype,data,num,db){
+      var y = '('+'?, '.repeat(num-1)+'?)';
+      console.log('insert data: ',data);
+      return db.executeSql(`INSERT INTO ${doctype} VALUES ${y}`,data).then(res => {
+        console.log("Inserted entry!!!");
+        return res;
+      })
+      .catch(err => {
+        console.log('error: ', err);
+      });
+    }
+
+    getAll(doctype,condtion,db){
```

Comment:
you can't change the function signature of `getAll` it should only take parameters as provided by the base Database class